Makefile

```diff
@@ -753,6 +760,50 @@ set-manifest-image:
 	$(info Updating kustomize image patch file for manager resource)
 	sed -i'' -e 's@image: .*@image: '"${MANIFEST_IMG}:$(MANIFEST_TAG)"'@' $(TARGET_RESOURCE)
 
+## --------------------------------------
+## vm-operator
+## --------------------------------------
```

Comment:
We could also move this targets to a separate Makefile in `hack/tools/vm-operator`. The build + push targets for the operator are also located there.